doc/developers/contributing.rst

```diff
@@ -1325,6 +1325,27 @@ interface might be that you want to use it together with model evaluation and
 selection tools such as :class:`model_selection.GridSearchCV` and
 :class:`pipeline.Pipeline`.
 
+Setting `generate_only=True` allows for checks to be disassembled when
```

Comment:
I feel like *disassembled* is a technical term that most won't understand (I didn't).

Maybe use the same kind of sentence as in the docstring